src/assets/i18n/en.json

```diff
@@ -1162,4 +1162,7 @@
   "JOYFUL_LEARNING_TEXT": "For joyful learning, we have redesigned our platform making it easier for you to find content",
   "JOYFUL_WARNING_TEXT": "If you want to change the theme, go to your menu and switch theme",
   "LETS_GO": "Let's go"
+  "SET_PREFERENCE": "Set your preference",
+  "OPEN_WITH": "Open With",
+  "OPEN_CONTENT_WITH": "Open Content With"
```

Comment:
Rename the keys to "FRMELEMNTS_LBL_OPEN_WITH" and "FRMELEMNTS_LBL_OPEN_CONTENT_WITH"